WordPress/src/main/java/org/wordpress/android/ui/prefs/WPSwitchPreference.java

```diff
@@ -37,6 +44,16 @@ public WPSwitchPreference(Context context, AttributeSet attrs) {
                 mHint = array.getString(index);
             } else if (index == R.styleable.SummaryEditTextPreference_iconTint) {
                 mTint = array.getResourceId(index, R.color.neutral);
+            } else if (index == R.styleable.SummaryEditTextPreference_switchThumbTint) {
+                mThumbTint = array.getColorStateList(index);
+            } else if (index == R.styleable.SummaryEditTextPreference_switchTrackTint) {
+                mTrackTint = array.getColorStateList(index);
+            } else if (index == R.styleable.SummaryEditTextPreference_preferenceTextColor) {
+                mTextColor = array.getResourceId(index, R.color.white);
```

Comment:
We actually pulling this color from one of the libraries, so if it'll get removed at some point we might be in trouble. I suggest using `android.R.color.white` instead (in other places too).

Reply:
ah, I didn't know that, good point 👍